src/apps/vpn/translations/strings.yaml

```diff
@@ -775,15 +775,6 @@ subscriptionManagement:
   addFirefoxRelay:
     value: Add Firefox Relay
     comment: Block of text appearing above a link to learn more about the VPN / Relay bundle offering.
-  valueProp1:
-    value: No logging of your network activity
-    comment: Value proposition used to promote the product
-  valueProp2:
-    value: No one can track you
-    comment: Value proposition used to promote the product
-  valueProp3:
-    value: 500+ servers in 30+ countries
-    comment: Value proposition used to promote the product
```

Comment:
We may use these strings at some point in the future, does it make sense to remove them now, and only add them back if we need them again later? Would they have to be translated again if we remove them and add them back in the future?

Reply:
I have a different question: were they ever used?

In general, we shouldn't land strings without code behind, because then we find ourselves in these situations (and already wasted time localizing them).

Please remove them, it's unlikely that we'll use them as is.

Reply:
They were being used in [ViewSubscriptionNeededIAP.qml](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6486/files#diff-f64cafb5944238cc6a0ddc325f30d14d6c4321eff3668730038097090eb43da8)